diversity.md

```diff
@@ -25,3 +25,11 @@ mentor and the awardee and hope everyone will have a great opportunity to learn
 from it.
 
 <img class="img-responsive feature-image" src="{{ site.baseurl }}/img/posts/cod.jpg" style="display:none">
+<div class="text-center">
```

Comment:
add  style="margin-top: 20px"

So it becomes <div class="text-center" style="margin-top: 20px">